mythril/analysis/module/modules/deprecated_ops.py

```diff
@@ -1,85 +0,0 @@
-"""This module contains the detection code for deprecated op codes."""
-
-from mythril.analysis.report import Issue
-from mythril.analysis.swc_data import DEPRECATED_FUNCTIONS_USAGE
-from mythril.analysis.module.base import DetectionModule, EntryPoint
-from mythril.laser.ethereum.state.global_state import GlobalState
-import logging
-
-log = logging.getLogger(__name__)
-
-DESCRIPTION = """
-Check for usage of deprecated opcodes
-"""
-
-
-class DeprecatedOperations(DetectionModule):
-    """This module checks for the usage of deprecated op codes."""
-
-    name = "Usage of deprecated instructions"
-    swc_id = DEPRECATED_FUNCTIONS_USAGE
-    description = DESCRIPTION
-    entry_point = EntryPoint.CALLBACK
-    pre_hooks = ["ORIGIN", "CALLCODE"]
```

Comment:
We might miss the CALLCODE if this is removed, right?

Reply:
Yes, but the CALLCODE is detection is useless anyway (I haven't even seen this triggered once) and if we wanted to search for deprecated instructions/call types we'd have to include all of them. Using Mythril for this is overkill IMO, simple static Solidity code analysis can do this more efficiently.